### PR TITLE
SUP-7072: fix failing check for ingest-attachment plugin on aws es se…

### PR DESCRIPTION
**Pull request by elbird**
…rvice

### CODE REVIEW

File: elasticsearch/src/main/java/com/gentics/mesh/search/impl/ElasticSearchProvider.java [hunk 1]
```diff
@@ -289,28 +293,21 @@ public Completable refreshIndex(String... indices) {
 
 	public boolean hasPlugin(String name) {
```
Comment (philippguertler) [inline elasticsearch/src/main/java/com/gentics/mesh/search/impl/ElasticSearchProvider.java]: Throw away this method, it is never used.

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
 		<spring.security.version>4.2.3.RELEASE</spring.security.version>
 		<ferma.version>2.4.0</ferma.version>
 		<ferma.orientdb.version>2.3.1</ferma.orientdb.version>
-		<elasticsearch.client.version>0.5.0</elasticsearch.client.version>
+		<elasticsearch.client.version>0.5.2-SNAPSHOT</elasticsearch.client.version>
```
Comment (philippguertler) [inline pom.xml]: Revert this

File: elasticsearch/src/main/java/com/gentics/mesh/search/impl/ElasticSearchProvider.java [hunk 3]
```diff
@@ -289,28 +293,21 @@ public Completable refreshIndex(String... indices) {
...
 		Objects.requireNonNull(name, "A valid plugin name must be specified.");
-		return registerdPlugins.contains(name);
+		return registeredPlugins.contains(name);
 	}
 
 	@Override
 	public Single<Set<String>> loadPluginInfo() {
```
Comment (philippguertler) [inline elasticsearch/src/main/java/com/gentics/mesh/search/impl/ElasticSearchProvider.java]: Remove this method

**Comment by elbird** [comment]
runnow

**Pull request merged**